### DISCUSSION TITLE
Passage à la licence CC-By-NC-ND

### DISCUSSION THREAD

**Discussion by hrenaud** [cnumr/best-practices-wordpress#27] [Q&A] [answered]
@bordage Suite à ton mail à @dNicolle , j'ai actualisé la licence de ce repo.

Tu peux la trouver ici :
https://github.com/cnumr/fiches-wordpress-greenit/blob/500248bc175f42e456eee86ecf2497b5c54e22a5/LICENSE

J'ai deux questions

- **Q1**. J'ai changé un peu le contenu pour matcher le sujet de ce repo, est-ce bon pour toi ?
- **Q2**. Est-ce que ce type de licence nous permet d'ouvrir d'ores et déjà le repo, et ainsi le passer en public ou bien, on attend que tout soit achevé/validé ?

**Comment by dNicolle** [comment]
@hrenaud est-ce qu'on peut la rendre visible sur le site ?

**Reply by hrenaud** [reply]
Elle l'est. Tu peux y accéder à droite quand tu arrives sur le repo.

**Comment by dNicolle** [comment, marked as answer]
Je parlais du site Gatsby.
Pas du repo :)
